src/org/usfirst/frc2175/subsystem/drivetrain/DrivetrainSubsystem.java

```diff
@@ -13,12 +13,14 @@
 import edu.wpi.first.wpilibj.PIDSourceType;
 import edu.wpi.first.wpilibj.RobotDrive;
 import edu.wpi.first.wpilibj.Talon;
+import edu.wpi.first.wpilibj.interfaces.Gyro;
 
 public class DrivetrainSubsystem extends BaseSubsystem {
     private Talon leftDriveTalon;
     private Talon rightDriveTalon;
     private Encoder leftDriveEncoder;
     private Encoder rightDriveEncoder;
+    private Gyro gyro;
```

Comment:
Let's start writing tests, even simple starter tests, for these classes.  They will also highlight problems like this - runtime ones.  It's much faster, easier, cheaper, better to find them now with some simple tests than at deploy and running the robot time.